firmware/application/bitmap.hpp

```diff
@@ -2531,6 +2531,44 @@ static constexpr Bitmap bitmap_icon_batt_icon{
     {16, 16},
     bitmap_icon_batt_icon_data};
 
+static constexpr uint8_t bitmap_icon_temperature_data[] = {
```

Comment:
I apologize for my misleading, but these two icons should be removed from the png folder and re-generate again, cuz they are not in use anymore.
I think linker would drop them anyway but only for clearer view, but don't worry let me do this in next PR.